uicomponent-compose/feed/src/main/java/io/github/droidkaigi/feeder/feed/BackLayerContent.kt

```diff
@@ -29,51 +33,73 @@ fun BackLayerContent(
     filterState: Filters,
     onFavoriteFilterChanged: (filtered: Boolean) -> Unit,
 ) {
-    Column {
-        Spacer(Modifier.height(16.dp))
-        Input(
-            text = if (filterState.filterFavorite) "Favorites" else "All",
-            onClick = { onFavoriteFilterChanged(filterState.filterFavorite.not()) }
-        )
-        Spacer(Modifier.height(8.dp))
+    Column(
+        modifier = Modifier
+            .padding(vertical = 12.dp)
+    ) {
+        FilterState.values().forEach { filter ->
```

Comment:
This is correct in design, but it's different from the behavior of checkboxes, so I talked to the designer before creating the issue.
I want to remove the All button and leave only Favorites so that it can be used as a checkbox.
However, it is not necessary to do it with this PR.